webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -126,7 +141,7 @@ class ManageManifestModal extends Component {
                     <FormControl
                       id="cdnUrl"
                       type="text"
-                      value={organization.redhat_repository_url}
+                      value={organization.redhat_repository_url || ''}
```

Comment:
if `organization.redhat_repository_url` is undefiend a warning will appear because it's a controlled input.